public/index.html

```diff
@@ -18,107 +22,292 @@
   <header role="banner" class="header">
     <div class="header__wrapper">
       <h1 class="header__primary">
-        <span class="header__primary--top">A Viget Exploration</span>
-        <img class="header__primary--btm icon" src="./src/icons/title.svg">
+        <span class="header__primary--top hdr-text">A Viget Exploration</span>
+        <img class="header__primary--btm" src="assets/images/title.svg" alt="improving medical adherence">
       </h1>
-
-      <img class="header__text-box__pills1 icon" src="src/icons/viget-logo.svg">
-      <img class="header__text-box__pills2 icon" src="src/icons/yellow-pill.svg">
-      <img class="header__text-box__pills3 icon" src="src/icons/more-pills.svg">
+      <img class="header__text-box__pills1" src="assets/images/viget-logo.svg" alt="pills">
+      <img class="header__text-box__pills2" src="assets/images/yellow-pill.svg" alt="a pill">
+      <img class="header__text-box__pills3" src="assets/images/more-pills.svg" alt="pills">
```

Comment:
I think these decorative images are a good example of when to use `alt=""` and have a screen reader ignore them. They add illustration flare, but not any real content.